cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestConstants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.engine.api.service;
+
+public class CohortEngineRestConstants {
+	public static final String SERVICE_MAJOR_VERSION = "v1";
+	public static final String SERVICE_TITLE = "IBM Cohort Engine";
+	public static final String SERVICE_DESCRIPTION = "Service to evaluate cohorts and measures";
+
+	// Provide a set of package names for swagger to search for REST APIs (comma delimited).
+	// Static assignment example
+	//public static final String SERVICE_SWAGGER_PACKAGES =  "com.ibm.watson.common.service.package1, com.ibm.watson.common.service.package2";
```

Comment:
Should this be removed as the variable is configured on line 19?

Reply:
It could be, but it is part of the comment illustrating of how to use include multiple packages using a comma separated list and I think contains useful information.